### ISSUE TITLE
BUG: Agents not displaying updated API Key name

### ISSUE THREAD

**Issue by TaliaSRTR** [PrefectHQ/ui#826]
**Bug Description**

**Details**
- Running agents with prefect `0.14.5`
- Using `systemd` to keep agents alive

**Steps To Reproduce**

1. Have a preexisting agent running against your cloud instance using an API token
2. Create a Service Account and generate an API Key for it
3. Stop the agent and immediately restart it with all the same parameters (e.g. agent type, labels, etc.) that you start it up with the first time, except replace the old API token you were passing in to the `--token` argument with the new API key
4. Observe that the UI is still showing the agent as using the same token/key under the "TOKEN NAME"

**Browsers Tested:**
* [ ] Chrome
* [x] Firefox
* [ ] Safari
* [ ] Edge
* [ ] IE

----------
**Notes**

**Comment by znicholasbrown** [comment]
Hi @TaliaSRTR - thanks for opening this! This looks to be a bug on the Cloud side so I've raised this internally. I'll leave this open until the issue is resolved 😄

**Comment by TaliaSRTR** [comment]
Okay good to know @znicholasbrown ; just to be sure, does this mean the issue can only be resolved by eventually upgrading the version of Prefect I'm running? Or is this something that could be fixed without my needing to touch anything (except perhaps to relaunch the agents)?

**Comment by znicholasbrown** [comment]
Nope, when this is fixed it'll just work, no actions necessary on your part 😄
